docs/en/observability/profiling-get-started.asciidoc

```diff
@@ -66,16 +66,30 @@ Even if you're profiling a smaller fleet, we recommend configuring at least two
 
 To set up Universal Profiling on your {ecloud} deployment, you need to:
 
-- Enable the Universal Profiling app in Kibana
-- Configure data ingestion
+. <<profiling-enable-kibana, Enable Universal Profiling in {kib}>>
+. <<profiling-configure-data-ingestion, Configure data ingestion>>
 
 [discrete]
 [[profiling-enable-kibana]]
 === Enable Universal Profiling in Kibana
 
-You'll receive instructions on enabling Universal Profiling at the email you used in your https://docs.google.com/forms/d/e/1FAIpQLSd-SWVgvhO7Z_jAfaV9_bFGa0dUZPuX0JORzPGS8SDP7G-dVQ/viewform[sign-up form].
+To enable Universal Profiling in {kib}, complete the following steps:
 
-Once enabled, Universal Profiling in Kibana is located under *Observability* in the navigation menu.
+. Log in to https://cloud.elastic.co[{ecloud}] and locate your deployment.

```

Comment:
There was a big push a while back to use attributes when linking to the cloud console so that we can track documentation referrals. Not sure if we're still concerned about it, but it can't hurt.

```suggestion
. Log in to {ess-console}[{ecloud}] and locate your deployment.

```